wcfsetup/install/files/lib/data/package/update/server/PackageUpdateServerAction.class.php

```diff
@@ -43,4 +44,16 @@ class PackageUpdateServerAction extends AbstractDatabaseObjectAction implements
 	 * @inheritDoc
 	 */
 	protected $requireACP = ['create', 'delete', 'toggle', 'update'];
+	
+	/**
+	 * @inheritDoc
+	 */
+	public function validateDelete() {
+		parent::validateDelete();
+		
+		/** @var PackageUpdateServer $updateServer */
```

Comment:
Is the type hint really necessary? Due to the `mixin` comment in `PackageUpdateServerEditor` and the `method` comment here for `getObjects()`, this comment should be superfluous.